a8cteam51/sift-decisions
# Add test capability.

#### Changes proposed in this Pull Request

* Adds testing through `wp-env` (and adds `wp-env` to node packages).

#### Testing instructions



* make sure you run `composer install --dev` and `npm i`
* `npx wp-env start`
* `npm test` should run the one test.
